include/quic/crypto.hpp

```diff
@@ -17,6 +17,7 @@ namespace oxen::quic
     {
       public:
         virtual std::unique_ptr<TLSSession> make_session(const ngtcp2_crypto_conn_ref& conn_ref, bool is_client) = 0;
+        virtual ~TLSCreds() = default;
```

Comment:
I was about to say the same applies to `TLSSession` but then I expanded context here and saw that I did put one with a trace log to know when it happens...